doc/cygnus-ngsi/installation_and_administration_guide/performance_tips.md

```diff
@@ -36,6 +37,22 @@ Nevertheless, as explained above, it is highly recommended to increase at least
 
 [Top](#top)
 
+###<a name="section1.2"></a>Retries
+Batches could not be persisted. This is someting may occur from time to time because the sink is temporarilly not available, or the communications are failing. In that case, Cygnus has implemented a retry mechanism.
```

Comment:
I would propose to use 'may' or 'might' and not 'could': Batches *may* not be persisted
Typo: This is something that may occur ...
And: temporarilly -> temporarily